types/framework/entities/actor.d.ts

```diff
@@ -411,8 +408,8 @@ declare class Actor<
    * @param options - Item deletion options
    * @returns A Promise resolving to the deleted Owned Item data
    */
-  deleteOwnedItem(itemId: string, options?: any): Promise<ActiveEffect | Actor.OwnedItemData<D>>;
-  deleteOwnedItem(itemId: string[], options?: any): Promise<Array<ActiveEffect | Actor.OwnedItemData<D>>>;
+  deleteOwnedItem(itemId: string, options?: any): Promise<Actor.OwnedItemData<D>>;
+  deleteOwnedItem(itemId: string[], options?: any): Promise<Array<Actor.OwnedItemData<D>>>;
```

Comment:
Could we update the options here with `Entity.DeleteOptions`?

Reply:
If that's the correct thing to do, sure. I only wanted to change things related to `ActiveEffect`s here but I can make this adjustment if you want.

Reply:
done